LabelGenerator.py

```diff
@@ -264,7 +264,7 @@ def draw_resistor_stripe_border(c: Canvas, x: float, y: float, width: float, hei
     c.rect(x, y, width, height, fill=0, stroke=1)
 
 
-def draw_resistor_stripe(c: Canvas, x: float, y: float, width: float, height: float, stripe_value: int) -> None:
+def draw_resistor_stripe(c: Canvas, x: float, y: float, width: float, height: float, stripe_value: int, tolerance_color = "grey") -> None:
```

Comment:
`stripe_value` already controls the color and should be used instead of adding a new `tolerance_color` argument